Code/GraphMol/RGroupDecomposition/Wrap/rdRGroupComposition.cpp

```diff
@@ -306,6 +306,9 @@ struct rgroupdecomp_wrapper {
         .def_readwrite("removeAllHydrogenRGroupsAndLabels",
                        &RDKit::RGroupDecompositionParameters::
                            removeAllHydrogenRGroupsAndLabels)
+        .def_readwrite("allowMultipleRGroupsOnUnlabelled",
```

Comment:
@bp-kelley I would really keep the default to `false` for backwards compatibility, otherwise existing code might break badly.

Reply:
I agree with Paolo, although for slightly different reasons.  I think this is the right way to do RGD, but I'd like to have the space to see how it performs on more and larger data sets without it being the default.  There may be performance issues as it is going to be a lot more combinatorial for saturated cores.  It would be good to iron out all the issues before considering making it the default.

Reply:
That's a convincing argument.  At some point I'm inclined to make the API simpler and do the right thing, especially with the current interactions with molzip and freewilson.

Reply:
> That's a convincing argument. At some point I'm inclined to make the API simpler and do the right thing, especially with the current interactions with molzip and freewilson.

I agree that it's worth thinking about a simplified API here that covers the 90% use case. Of course, I'm not sure if that's the same API as the one for molzip and freewilson. ;-)